lib/glific/partners/saas.ex

```diff
@@ -30,7 +30,8 @@ defmodule Glific.Partners.Saas do
           organization: Organization.t() | Ecto.Association.NotLoaded.t() | nil,
           phone: String.t() | nil,
           inserted_at: :utc_datetime | nil,
-          updated_at: :utc_datetime | nil
+          updated_at: :utc_datetime | nil,
+          json: map() | nil
```

Comment:
I don't think json makes any sense here. 
Let's make it isv_credentials which will be a jsonb column
```suggestion
          isv_credentials: map() | nil
```

Reply:
made requested changes